scripts/gendocs/generate

```diff
@@ -40,7 +40,6 @@ trap cleanup EXIT
 # Create fake GOPATH
 echo "+++ Creating temporary GOPATH"
 export GOPATH="${tmpdir}/go"
-export GOPROXY="https://proxy.golang.org"
```

Comment:
It was probably added before this became the default
> For versions earlier than Go 1.13, you can configure the go command to download modules using this module mirror by setting GOPROXY=https://proxy.golang.org.
-- https://proxy.golang.org/#EnvironmentSetup